src/test/scala/mesosphere/marathon/core/launchqueue/impl/TaskLauncherActorTest.scala

```diff
@@ -286,103 +303,52 @@ class TaskLauncherActorTest extends AkkaUnitTest {
       assert(counts.inProgress)
       assert(counts.finalInstanceCount == 1)
       assert(counts.instancesLeftToLaunch == 1)
-
-      Mockito.verify(instanceTracker).instancesBySpecSync
-      val matchRequest = InstanceOpFactory.Request(f.app, offer, Map.empty, additionalLaunches = 1)
-      Mockito.verify(instanceOpFactory).matchOfferRequest(matchRequest)
-      verifyClean()
-    }
-
-    "Process task launch timeout" in new Fixture {
```

Comment:
Seems like we need this test :)

Reply:
See the test is just calling `assert(scheduleCalled)`. I tests an implementation details not a behaviour 🙁 .

Reply:
#6340 introduces a proper test.